static/js/ducks/candidates.js

```diff
@@ -1,12 +1,19 @@
 import messageHandler from "baselayer/MessageHandler";
 
 import * as API from "../API";
+import fetchCandidate from "./candidate";
 import store from "../store";
 
 export const FETCH_CANDIDATES = "skyportal/FETCH_CANDIDATES";
 export const FETCH_CANDIDATES_OK = "skyportal/FETCH_CANDIDATES_OK";
 export const FETCH_CANDIDATES_FAIL = "skyportal/FETCH_CANDIDATES_FAIL";
 
+export const FETCH_CANDIDATE_AND_MERGE = "skyportal/FETCH_CANDIDATE_AND_MERGE";
+export const FETCH_CANDIDATE_AND_MERGE_OK =
+  "skyportal/FETCH_CANDIDATE_AND_MERGE_OK";
+
+export const REFRESH_CANDIDATE = "skyportal/REFRESH_CANDIDATE";
```

Comment:
does this need a corresponding `REFRESH_CANDIDATE_OK`?

Reply:
nope, as this action originates from the backend, not the frontend, and is handled by the websocket messagehandler.